perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/CommitToBranchSimulationSameTable.scala

```diff
@@ -27,8 +27,9 @@ import org.projectnessie.perftest.gatling.Predef.nessie
 
 import scala.concurrent.duration.{FiniteDuration, HOURS, NANOSECONDS, SECONDS}
 
-/** Gatling simulation to perform commits against Nessie. Has a bunch of
-  * configurables, see the `val`s defined at the top of this class.
+/** Gatling simulation to perform commits against Nessie. Has a bunch of It
```

Comment:
something is missing here in `Has a bunch of It`

Reply:
please see next line

Reply:
yes, the entire sentence is `Has a bunch of It commits the data...`, which seems that there's something missing